### ISSUE TITLE
Capture parentage of graphics calls

### ISSUE THREAD

**Issue by MKLau** [ProvTools/provR#62]
This is currently being worked on by B Lerner in RDataTracker.

**Comment by blernermhc** [comment]
I'm not sure what this is referring to.  The development version of RDT supports this.  Are there specific things missing from that?

**Comment by MKLau** [comment]
If this is implemented, that's great. Perhaps this has changed after I last tested. Previously, provenance was not monitoring the graphics device, but recording specific functions known to be involved in plotting (e.g. dev.off). If it's still the case that provenance is not being captured by monitoring their graphics environment, then I think we should implement this.

**Comment by blernermhc** [comment]
You are right that the current implementation looks at the function calls.  It looks for every function in the default plotting package and could easily be extended to support ggplot2 or other plotting packages.

I do not know how to track the graphics device itself.  I think the actual plotting is done in C code.

**Comment by MKLau** [comment]
This package (called recordr, but different from the one developed by NCEAS) that I've been reviewing is aimed at tracking R graphics: https://github.com/gmbecker/recordr

I haven't dug down into their code yet.

**Comment by blernermhc** [comment]
Looks interesting and non-trivial.  I think it would be good to run their vignette to see what kind of output they get before diving in to try to understand their code.

https://github.com/gmbecker/recordr/blob/master/vignettes/index.Rmd

Is this the project that CRAN rejected?
